src/command.js

```diff
@@ -302,6 +338,21 @@ const command = {
 		});
 	},
 
+	reOrderBumpAndLocalizationCommits({ onError } = {}) {
```

Comment:
I'd rename this to something like `rebaseBumpToLatestCommit" or something like that.

As it is now it feels too similar to `reOrderLatestCommits` which returns a string, but this method does much more than that and it doesn't necessarily have to do anything with localization anymore.